ci-operator/config/openshift/osde2e/openshift-osde2e-master.yaml

```diff
@@ -0,0 +1,54 @@
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.11
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: verify
+  artifact_dir: /tmp/artifacts
+  commands: |
+    set -o pipefail
+    export HOME=$(mktemp -d)
+    mkdir -p /tmp/artifacts
+
+    go get -u github.com/Masterminds/glide && glide install --strip-vendor
```

Comment:
If you add this to your `binary_build_commands` or `test_binary_build_commands` you can then do `from: bin` or `from: test-bin`, respectively, on these two tests, and you will re-use the `go get` steps instead of doing them on-demand every time.

Reply:
I had initially been doing this successfully, however for whatever reason this ceased working for me:

```
Replaced Dockerfile FROM image pipeline:src

Pulling image docker-registry.default.svc:5000/ci-op-bs39d79q/pipeline@sha256:76c44cb9bf3aac75a20370cd8d41b6982dd4295efa4030c30b9b938cb05e1372 ...
Pulled 3/3 layers, 100% complete
Extracting
--> FROM docker-registry.default.svc:5000/ci-op-bs39d79q/pipeline@sha256:76c44cb9bf3aac75a20370cd8d41b6982dd4295efa4030c30b9b938cb05e1372 as 0
--> RUN ["/bin/bash","-c","set -o errexit; umask 0002; go get -u github.com/Masterminds/glide \u0026\u0026 glide install --strip-vendor"]
[0;31m[ERROR]	[mFailed to find glide.yaml file in directory tree: Cannot resolve parent of /
error: build error: running '/bin/bash -c set -o errexit; umask 0002; go get -u github.com/Masterminds/glide && glide install --strip-vendor' failed with exit code 2
2019/10/03 19:22:38 Ran for 1m9s
2019/10/03 19:22:38 Submitted failure event to sentry (id=503421d2fff54d20af6b14a18b3d3dc5)
error: could not run steps: step bin failed: could not wait for build: the build bin failed after 23s with reason DockerBuildFailed: Docker build strategy has failed.

Extracting
--> FROM docker-registry.default.svc:5000/ci-op-bs39d79q/p...3aac75a20370cd8d41b6982dd4295efa4030c30b9b938cb05e1372 as 0
--> RUN ["/bin/bash","-c","set -o errexit; umask 0002; go ...sterminds/glide \u0026\u0026 glide install --strip-vendor"]
[0;31m[ERROR]	[mFailed to find glide.yaml file in directory tree: Cannot resolve parent of /
error: build error: running '/bin/bash -c set -o errexit; ...de && glide install --strip-vendor' failed with exit code 2
```

Running `HOME=$(mktemp -d)` before this seemed to solve the issue, but I'm not 100% convinced that was the fix, it just happened to work.

Reply:
I'll give this another shot, though.

Reply:
Looks like it's working now. Not sure why, but yay! Resolving.